docs/gh-pages-build.js

```diff
@@ -0,0 +1,142 @@
+#!/usr/local/bin/node
+/**
+ * Build a release for the gh-pages docs site.
+ */
+
+var fs = require('fs');
+var execSync = require('child_process').execSync;
+
+var usage = '\nbuild <vn.n.n[-pre[.n]] | HEAD> [-p]\n';
+var versionsFile = './src/www/versions.json';
```

Comment:
@mbrookes Always try resolving these relative paths using `path.resolve(__dirname, './src/...')` because relative paths are reolved relative to current working directory, so if someone does this: `node docs/gh-pages-build.js` from the root directory this won't work.